Backend/app/dashboard/CommonAPIs/spec/ProviderPlatform/Management/API/Ride.yaml

```diff
@@ -21,7 +21,6 @@ apis:
         rideShortId: ShortId Ride
         customerPhoneNo: Text
         driverPhoneNo: Text
-        fareDiff: HighPrecMoney
```

Comment:
why its removed? don't used by frontend?

Reply:
yeah that is just making query bad in terms of perf.